src/lib/viewers/media/MP3Viewer.js

```diff
@@ -52,14 +52,18 @@ class MP3Viewer extends MediaBaseViewer {
 
         this.controls.render(
             <MP3Controls
+                autoplay={this.cache.get('media-autoplay')}
```

Comment:
Does this work as expected if these values aren't defined? I believe we set defaults for other values.